test/local/push.js

```diff
@@ -316,6 +316,70 @@ describe('push', () => {
   )
 
   it(
+    'sendPush pushes to ios >=9.0 devices if it is triggered with a "device connected" command',
+    () => {
+      var data = Buffer.from(JSON.stringify({command: 'fxaccounts:device_connected'}))
```

Comment:
I vaguely remember a conversation where we said new code would try to use ES6 and const/let where possible?